core/commands/repo.go

```diff
@@ -36,6 +42,7 @@ var RepoCmd = &cmds.Command{
 
 	Subcommands: map[string]*cmds.Command{
 		"stat": repoStatCmd,
+		"has":  repoHasCmd,
```

Comment:
Imo this doesn't fit into `ipfs repo`. `ipfs block` or `ipfs dag` would be a better place (I'm not sure which one is better)

Reply:
I choose `ipfs repo` because we are interested about what is inside the repo and nothing else. The question is 'is this stored on my node ?'.

Both `ipfs block` and `ipfs dag` will do network request and treat the graph of object as something that exist somewhere, regardless of the repo state.